students/alyssiamaiorano.json

```diff
@@ -0,0 +1,15 @@
+{
+  "firstName": "Alyssia",
+  "lastName": "Maiorano",
+  "nickName": "alyssia",
+  "privateName": false,
+  "picture": "/c/:/Desktop/Digital-History/assignment-00-git-and-github/images/raven_claw.JPG",
```

Comment:
You can't use the absolute path like this, because it will differ from computer to computer. You need to use the relative path, `images/raven_claw.jpg.JPG`. This likely wouldn't have worked on yr computer (since the image name isn't quite righ), did you try running collectjson?